### PR TITLE
MERC-409: Utilize standard import style

### PR DESCRIPTION
**Pull request by hegrec**
Remove JSPM and its style of imports.

Use webpack for generation of a `window` bound bundle, but also support standard es6 module exports for use with dependency bundlers.

Supports ES6, global library export on the dist file, inclusion as a contained module, karma tests as before.

@mickr @mcampa

### CODE REVIEW

**Comment by mickr** [comment]
MRW I win the Democratic Primary in Michigan.

**Comment by mickr** [comment]
http://i.imgur.com/8YLIbIl.gif

**Comment by hegrec** [comment]
Ready for review, this does work with the existing theme, just minor changes required to manually resolve JSPM dep tree.

File: package.json [hunk 1]
```diff
@@ -1,44 +1,34 @@
 {
   "name": "@bigcommerce/stencil-utils",
   "version": "0.4.0",
-  "main": "dist/stencil-utils.min.js",
+  "main": "src/main.js",
```
Comment (mcampa) [inline package.json]: I think we need to commit the built and point this to `dist/main.js`

Comment (hegrec) [inline package.json]: Talked offline, the dist file still exists at dist/stencil-utils.min.js, but the main now points to the es6 source code for standard module imports.

dist bundle can still be dropped in as a `<script>` tag on the markup.

**Comment by mcampa** [comment]
:+1:  LGTM

**Pull request merged**